modules/learn/pages/data/expiration.adoc

```diff
@@ -9,93 +10,122 @@
-After its expiration time is reached, the item will be deleted by Couchbase Server.
-Deleted items and their contents are no longer available: access-attempts are treated as if the items never existed.
+The `maxTTL` setting is not available for scopes. 
 
-The maximum value for TTL is MAX32INT seconds (2147483648, or 68.096 years).
-The default value is 0, which indicates that TTL is disabled.
-If TTL is changed from the default, it is thereby enabled.
+== Expiration Setting Priorities
 
-Whenever TTL is modified, items that existed _prior_ to the modification:
+In general, non-zero expiration set at a lower level take precedence over settings at a higher level--non-zero expiration times set at a document level override a collection's non-zero `maxTTL` settings, which in turn override `maxTTL`settings at the bucket level. 
+The three exceptions to this rule are:
 
-* Remain subject to the previous TTL, if TTL was previously enabled.
-* Remain subject to no TTL, if TTL was not previously enabled.
+* As mentioned earlier, you cannot set a document's expiration to be longer than a non-zero `maxTTL` setting of the document's collection or bucket.
+If you try to set the document's expiration to be longer than its collection or bucket's `maxTTL` setting, Couchbase Server uses the `maxTTL` setting instead. 
 
-Items created or modified _following_ the modification of TTL are subject to the modified TTL.
+* Setting a collection's `maxTTL` to zero does not override the bucket's `maxTTL` setting. Instead, setting the collection's `maxTTL` to zero has it inherit the `maxTTL` setting from the bucket. 
 
-TTL can be established on Couchbase and Ephemeral buckets: it cannot be established on Memcached buckets.
-TTL can be established on collections: it cannot be established on scopes.
-Note that TTL is only available in the Enterprise Edition of Couchbase Server.
+* Setting a document's expiration to 0 does not prevent it from expiring if its collection or bucket has a `maxTTL` setting that's greater than zero. 
+This behavior means you cannot prevent a document from expiring if its collection or bucket has a `maxTTL` setting that's greater than zero. 
 
-[#expiration-bucket-versus-item]
-== Expiration: Bucket, Collection, and Item
+The following table summarizes the interaction between a document's expiration setting and the `maxTTL` settings of the collection and bucket that contain it.
 
-As described in xref:java-sdk:howtos:kv-operations.adoc#document-expiration[Document Expiration], an expiration time can be specified per document, by means of Couchbase SDK APIs: this is referred to as the item’s TTL.
+[cols="1,1,1,1"]
+|===
+| Document Expiration Setting | Collection `maxTTL` | Bucket `maxTTL` | Effect
 
-In cases where an item has a TTL, the item's TTL may need to be modified, depending on the TTL of either its bucket or its collection:
+| Null, unset, or 0
```

Comment:
What's the difference between "Null" and "0"? How does one set to "Null"?  Shouldn't "Unset or 0" suffice?

Reply:
I'm not sure where I picked that terminology up. I'll switch it to unset or 0.